include/uv.h

```diff
@@ -1292,8 +1292,10 @@ enum {
   UV_THREAD_PRIORITY_LOWEST = -2,
 };
 
+#if !defined(__MVS__)
 UV_EXTERN int uv_thread_getpriority(uv_thread_t tid, int* priority);
 UV_EXTERN int uv_thread_setpriority(uv_thread_t tid, int priority);
+#endif
```

Comment:
The symbols should always be available. You can make them return UV_ENOSYS to indicate it's not supported on zos. Update the thread_priority test to either call RETURN_SKIP or check for UV_ENOSYS.